device_query/device_query.py

```diff
@@ -87,27 +88,29 @@ def main(verify):
             }
             encoded_post_data = bytes(json.dumps(post_data, indent=4), "utf-8")
 
-            time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-            print(f"[{time}] Sending request...")
+            logging.info(f"Sending request...")
 
-            if verify is not None:
-                r = requests.post(server_url, headers=headers, data=encoded_post_data, verify=verify)
+            if args.verify is not None:
+                r = requests.post(server_url, headers=headers, data=encoded_post_data, verify=args.verify)
```

Comment:
I was thinking, if you set the default of `--verbose` to False, you can get rid of this `if`, couldn't you?

Reply:
You mean `--verify`, right? But yeah, could do this